app/models/vote.rb

```diff
@@ -0,0 +1,5 @@
+class Vote < ApplicationRecord
+  validates :vote, presence: true
```

Comment:
Probably not the best name for this attribute, since it is now `Vote#vote`. How about `Vote.value` or something like that.